data/datavalid.yml

```diff
@@ -11,6 +11,8 @@ files:
     schema: stop_and_search
   fuse/appeals.csv:
     schema: appeal_hearing
+  fuse/citizen.csv:
+    schema: citizen
```

Comment:
Like I said, please introduce `uof_officer` schema as well. this schema should be named `uof_citizen` instead of `citizen` because it is dependent on the use of force data.